client/src/Backend/Network/Blockchain.ts

```diff
@@ -1,5 +1,4 @@
-// These are loaded as URL paths by a webpack loader
-import diamondContractAbiUrl from '@darkforest_eth/contracts/abis/DarkForest.json';
+import diamondContractAbiUrl from '@darkforest_eth/contracts/abis/DarkForest.json?url';
```

Comment:
This is not relevant to this PR, but why do we need to fetch the ABIs instead of just directly loading the file? Is it for performance to just fetch as needed?

Reply:
This is to avoid including the massive ABI in the primary bundle. Loading my URL will allow the UI to render quicker and then we can bootstrap the ABI later.